README.md

```diff
@@ -1,31 +1,27 @@
+# Microsoft Azure SDK for .NET
 
+| Component | Build Status |
+| --------- | ------------ |
+| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
+| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-# Microsoft Azure SDK for .NET
- ----------
-The Microsoft Azure SDK for .NET allows you to build applications
-that take advantage of scalable cloud computing resources.
+Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It’s the freedom to build, manage, and deploy applications on a massive, global network using your favorite tools and frameworks.
+
+This repo contains the libraries to allow you to easily leverage Azure from your applications and tools.
```

Comment:
These changes are unexpected. Please revert this file to the remote master

Reply:
Pulling the master/readme.md and committing it.

Reply:
The file is an exact copy of azure/azure-sdk-for-net/master readme.md. I am not sure why the diff is showing up this way.

Reply:
How are you reverting (git checkout ?)

Reply:
git checkout upstream/master -- README.md
I ran this command, and it doesn't modify the file.

Reply:
Can you post what your upstream is here

Reply:
Upstream is the Azure/azure-sdk-for-net repo